### DISCUSSION TITLE
Add Example pictures of the available configurations

### DISCUSSION THREAD

**Discussion by laufersteppenwolf** [PlummersSoftwareLLC/NightDriverStrip#58] [Ideas]
As I was going through the available effects/configs, I was not always sure which config was meant for which application.
Maybe it wouldn't be a bad idea to add some kind of small documentation (maybe even just one or two pictures) for each config?

Thoughts?

**Comment by DanMan32** [comment]
I agree.   This project could use extensive documenting what does what and where to do it.
For example list all the defines like what has been done for a few of them in the Readme and where they could be set.  It took me a bit to figure out that in platformIO.ini, the build_flags prefixed with D set a #define.   So ddemo = 1 creates a #define DEMO = 1.

Not a complaint, just an idea we could start doing.
